jx_sqlite/snowflake.py

```diff
@@ -135,6 +141,8 @@ def _add_column(self, column):
         )
 
         self.add_column_to_schema(column)
+        if command:
+            return command
 
     def _nest_column(self, column, new_path):
```

Comment:
or may be we can rename those columns  or delete them by creating temporary table copying all columns but that one.